lib/howl/ui/icon.moon

```diff
@@ -0,0 +1,34 @@
+-- Copyright 2015 The Howl Developers
+-- License: MIT (see LICENSE.md at the top-level directory of the distribution)
+
+import style, StyledText from howl.ui
+
+icons = {}
+
+style_name = (icon_name) -> '_icon_font_'..icon_name
+
+define = (name, definition={}) ->
```

Comment:
Should we have a `define_default` here as well? Would tie in with the other registries.